plugins/webp-uploads/settings.php

```diff
@@ -40,6 +40,18 @@ function webp_uploads_register_media_settings_field(): void {
 			'show_in_rest' => false,
 		)
 	);
+
+	// Add a setting to generate fallback images in all sizes including custom sizes.
+	register_setting(
+		'media',
+		'perflab_generate_fallback_all_sizes',
```

Comment:
Maybe rename this to `perflab_generate_all_fallback_sizes`, that reads a bit better.